### ISSUE TITLE
Refactor codegen to work with Embroider

### ISSUE THREAD

**Issue by jenweber** [cardstack/cardstack#764]
> Module not found: Error: Can't resolve '@cardstack/plugin-utils/environment' in '/private/var/folders/9k/l7x9d45s373_nchwp07mkrqr0000gn/T/embroider/900d16/Users/jenweber/cardstack/cardstack/packages/codegen'

To-do:
- [x] change environment codegen to modules
- [x] change models codegen to modules
- [x] change ethereum codegen to modules
- [x] change github-auth codegen to modules
- [x] change live-queries codegen to modules
- [x] change test-support codegen to modules
- [x] ~~figure out how to test ethereum codegen (there were no node/ember tests for it)~~ removed
- [x] figure out how to test github-auth codegen (no test coverage of generated code) - added it to an existing test
- [x] test live-queries codegen. created new node-tests.
- [x] remove generateCode use

**Comment by jenweber** [comment]
Steps to show generated code:

From `packages/cli`, run:
```
CARDSTACK_DEV=true node ./bin/cardstack.js run
```

cd into the `tmp dir` printed to the console, then one more level into cardstack-standard-app. From there:

```
ember hub:start
```

Visit `http://localhost:3000/codegen/foo` to see generated code. Not all codegens are used by the CLI's app so some generators will not be visible, and instead you'll just need to run the tests.

**Comment by jenweber** [comment]
We removed the ethereum codegen because we would likely need to replace it in the future anyway.

**Comment by jenweber** [comment]
Closed by #768
